styles/mob.css

```diff
@@ -29,26 +29,30 @@
     flex-direction: column;
     justify-content: center;
     text-align: center;
+    padding-top: 1em;
+    padding-bottom: 1em;
 }
+    header a {
+      border: 0;
+    }

```

Comment:
You need to undo the top border on the un-hovered links as well in order to prevent the nav items from shifting up and down when hovering